integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDiagnosticsFailedCondition.java

```diff
@@ -0,0 +1,668 @@
+
+    logger.info("Assign a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace is null");
+    domainNamespace = namespaces.get(1);
+
+    // set the service account name for the operator
+    opServiceAccount = opNamespace + "-sa";
+
+    // install and verify operator with REST API
+    installAndVerifyOperator(opNamespace, opServiceAccount, true, 0, domainNamespace);
+
+    // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
+    // this secret is used only for non-kind cluster
+    createSecretForBaseImages(domainNamespace);
+
+    // create secret for admin credentials
+    logger.info("Create secret for admin credentials");
+    adminSecretName = "weblogic-credentials";
+    createSecretWithUsernamePassword(adminSecretName, domainNamespace,
+            ADMIN_USERNAME_DEFAULT, ADMIN_PASSWORD_DEFAULT);
+
+    // create encryption secret
+    logger.info("Create encryption secret");
+    encryptionSecretName = "encryptionsecret";
+    createSecretWithUsernamePassword(encryptionSecretName, domainNamespace,
+            "weblogicenc", "weblogicenc");
+  }
+
+  /**
+   * Test domain status condition with a bad model file.
+   * Verify the following conditions are generated after a introspector failure.
+   * type: Completed, status: true
+   * type: Available, status: false
+   * type: Failed, status: true
+   */
+  @Test
+  @DisplayName("Test domain status condition with bad model file")
+  void testBadModelFileStatus() {
+    // build a build with empty WebLogic domain
+    String imageName = MII_BASIC_IMAGE_NAME;
+    String imageTag = "empty-domain-image";
+    buildMIIandPushToRepo(imageName, imageTag, null);
+
+    String badModelFileCm = "bad-model-in-cm";
+    Path badModelFile = Paths.get(MODEL_DIR, "bad-model-file.yaml");
+    final List<Path> modelList = Collections.singletonList(badModelFile);
+
+    logger.info("creating a config map containing the bad model file");
+    createConfigMapFromFiles(badModelFileCm, modelList, domainNamespace);
+
+    try {
+      // Test - test bad model file status with introspector failure
+      logger.info("Creating a domain resource with bad model file from configmap");
+      Domain domain = createDomainResourceWithConfigMap(domainUid, domainNamespace, adminSecretName,
+          OCIR_SECRET_NAME, encryptionSecretName, replicaCount, imageName + ":" + imageTag, badModelFileCm, 30L);
+      createDomainAndVerify(domain, domainNamespace);
+
+      //check the desired completed, available and failed statuses
+      checkStatus("True", "False", "True");
+
+    } finally {
+      deleteDomainResource(domainUid, domainNamespace);
+      deleteConfigMap(badModelFileCm, domainNamespace);
+    }
+  }
+
+  /**
+   * Test domain status condition with replicas set to more than maximum size of the WebLogic cluster created.
+   * generated: type: Completed, status: true type: Available, status: false type: Failed status: true
```

Comment:
Maintain the same description format as previous test

 type: Completed, status: true
 type: Available, status: false
 type: Failed, status: true

Reply:
following same format for all tests